meson_options.txt

```diff
@@ -2,6 +2,6 @@ option('default_codec', type: 'string', value: 'libvpx-vp9', description: 'Codec
 option('default_framerate', type: 'integer', value: 60, description: 'Video framerate that will be used by default')
 option('default_audio_codec', type: 'string', value: 'libopus', description: 'Audio codec that will be used by default')
 option('default_audio_sample_rate', type: 'integer', value: 48000, description: 'Audio sample rate that will be used by default')
-option('default_container_format', type: 'string', value: 'mkv', description: 'Container file format that will be used by default')
+option('default_container_format', type: 'string', value: '"mkv"', description: 'Container file format that will be used by default')
```

Comment:
Wouldn't this break if someone sets a different option without quotes via meson?